webapp/src/App.tsx

```diff
@@ -44,19 +44,21 @@ const App = () => {
   }, [authLoginData, dispatch])
 
   return (
-    <ThemeProvider theme={theme}>
-      <BrowserRouter>
-        {routeNotFound ? (
-          <NotFound offsetHeight={0} />
-        ) : (
-          <Routes>
-            <Route index element={<Navigate to="dashboard" replace />} />
-            <Route path="dashboard/*" element={<Dashboard />} />
-            <Route path="*" element={<NotFound shouldRedirect={true} />} />
-          </Routes>
-        )}
-      </BrowserRouter>
-    </ThemeProvider>
+    <StyledEngineProvider injectFirst>
```

Comment:
Why is this required?

Reply:
This ensures that MUI styles don't override any custom styles we've defined. Without the injectFirst prop, MUI styles are added at the end of the <head> element so they may override any custom styles we've defined.

Reply:
Is that not the point of specifying an additional theme, to override the original MUI theme? Or am I missing something?

Reply:
Looks like this is necessary when running MUI themes with css styles... Unforfunate, but oh well. Hopefully we can migrate these CSS styles to MUI in the future